webapp/packages/plugin-navigation-tree/src/NavigationTree/ProjectsRenderer/NavigationNodeProjectControl.m.css

```
@@ -0,0 +1,36 @@
+.treeNodeControl {
+  transition: opacity 0.3s ease;
+  opacity: 1;
+
+  &.outdated,
+  .dragging {
```

Here is probably a mistake:
```suggestion
  &.outdated,
  &.dragging {
```